routers/web/repo/migrate.go

```diff
@@ -259,6 +259,15 @@ func setMigrationContextData(ctx *context.Context, serviceType structs.GitServic
 	ctx.Data["service"] = serviceType
 }
 
+func MigrateRetryPost(ctx *context.Context) {
+	if err := task.RetryMigrateTask(ctx.Repo.Repository.ID); err != nil {
+		log.Error("Retry task failed: %v", err)
+		ctx.ServerError("task.RetryMigrateTask", err)
+		return
+	}
+	ctx.Redirect(ctx.Repo.Repository.Link())
```

Comment:
The JS code handles "window.reload()", so here it doesn't need to redirect to anywhere.